src/NLog/Layouts/JsonLayout.cs

```diff
@@ -53,6 +53,7 @@ public class JsonLayout : Layout
         public JsonLayout()
         {
             this.Attributes = new List<JsonAttribute>();
+            this.RenderEmptyLiteral = true;
```

Comment:
I think it's wise to keep the behavior backwards-compatible (so `false` as default)